services/viva/microservice/src/lambdas/syncWorkflow.ts

```diff
@@ -120,14 +116,15 @@ export async function syncWorkflow(input: LambdaRequest, dependencies: Dependenc
 
   await Promise.all(
     caseListWithWorkflowId.map(async caseItem => {
+      const workflowId = caseItem.details.workflowId as string;
       const workflow = await dependencies.getWorkflow({
         personalNumber,
-        workflowId: caseItem.details.workflowId,
+        workflowId,
       });
 
       const caseKeys = { PK: caseItem.PK, SK: caseItem.SK };
-      await dependencies.updateCase(caseKeys, workflow.attributes);
-      await dependencies.syncWorkflowSuccess({ caseKeys, workflow: workflow.attributes });
+      await dependencies.updateCase(caseKeys, workflow);
+      await dependencies.syncWorkflowSuccess({ caseKeys, workflow });
```

Comment:
**Optional**
Since the requests aren't dependent on each other, we can do them in parallell. 

```js
await Promise.all([
  dependencies.updateCase(caseKeys, workflow),
  dependencies.syncWorkflowSuccess({ caseKeys, workflow })
]);
```